### PR TITLE
vit 2.0.0

### PR DESCRIPTION
**Pull request by arcnmx**
##### Motivation for this change

https://groups.google.com/forum/#!topic/taskwarrior-user/0dTbvFOOZY0

I'm not sure whether I should remove the previous `vit` package and replace the attr with this, or keep it around as `vit1`, or?

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @dtzWill as maintainer of the existing vit 1.3 package, also do you want to be on this version too?

### CODE REVIEW

**Comment by jonringer** [comment]
@worldofpeace thoughts on deprecation of pythonPackages.vit? It's currently marked broken.

**Comment by worldofpeace** [comment]
So vit 1.3 is in perl, but the latest is rewritten in python.

If vit has moved on from perl this is just a version update.

I see it as they're the same package, you can just update the `misc/vit` expression in place.
What versions of python it supports is just a detail of the new python code.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5465,6 +5467,8 @@ in {
 
   virtualenvwrapper = callPackage ../development/python-modules/virtualenvwrapper { };
 
+  vit = callPackage ../development/python-modules/vit { };
```
Comment (worldofpeace) [inline pkgs/top-level/python-packages.nix]: is there documented use of it as a  python module?

**Comment by jonringer** [comment]
>I included `meta.broken = python.isPy2` to indicate that it only supports python 3

you should use `disabled = isPy27` instead

**Comment by worldofpeace** [comment]
Thanks @arcnmx :sparkles:

**Pull request merged**